pkg/controller/migmigration/dvm.go

```diff
@@ -78,6 +80,24 @@ func (t *Task) getDirectVolumeMigration() (*migapi.DirectVolumeMigration, error)
 	return nil, nil
 }
 
+func (t *Task) getDirectVolumeMigrationProgressForDVM(dvm *migapi.DirectVolumeMigration) (*migapi.DirectVolumeMigrationProgress, error) {
+
+	dvmLabels := dvm.GetCorrelationLabels()
+	selector := labels.SelectorFromSet(dvmLabels)
+	dvmpList := migapi.DirectVolumeMigrationProgressList{}
+	err := t.Client.List(context.TODO(),
+		&k8sclient.ListOptions{
+			LabelSelector: selector,
+		},
+		&dvmpList)
+
+	if err != nil {
+		return nil, liberr.Wrap(err)
+	}
+
+	return &dvmpList.Items[0], nil
```

Comment:
Items can be empty I think this will panic as well

Reply:
Done, updated the code/